api/apps/users/model.coffee

```diff
@@ -45,38 +45,35 @@ bcrypt = require 'bcrypt'
   db.users.findOne { _id: ObjectId(id) }, (err, user) ->
     return callback err if err
     return callback null, user if user
-    async.parallel [
-      (cb) ->
-        request.get("#{ARTSY_URL}/api/v1/user/#{id}")
-          .set('X-Access-Token': accessToken).end cb
-      (cb) ->
-        request.get("#{ARTSY_URL}/api/v1/user/#{id}/access_controls")
-          .set('X-Access-Token': accessToken).end cb
-    ], (err, results) ->
-      return callback err if err
-      user = results[0].body
-      save user, accessToken, callback
+    request.get("#{ARTSY_URL}/api/v1/user/#{id}")
+      .set('X-Access-Token': accessToken)
+      .end (err, user) ->
+        return callback err if err
+        save user.body, accessToken, callback
 
 save = (user, accessToken, callback) ->
-  async.parallel _.compact([
+  async.parallel [
     (cb) ->
-      bcrypt.hash accessToken, SALT, cb
-    if user.type is "Admin"
-      (cb) ->
-        request
-        .get("#{ARTSY_URL}/api/v1/me/authentications")
+      request.get("#{ARTSY_URL}/api/v1/user/#{user.id}/access_controls")
         .set('X-Access-Token': accessToken).end cb
-  ]), (err, results) ->
+    (cb) ->
+      db.channels.find {user_ids: ObjectId(user.id)}, cb
+    (cb) ->
+      bcrypt.hash accessToken, SALT, cb
+  ], (err, results) ->
     return callback err if err
-    encryptedAccessToken = results[0]
+    user.partner_ids = _.map results[0].body, (partner) ->
+      partner.property._id
+    user.channel_ids = _.pluck results[1], '_id'
+    encryptedAccessToken = results[2]
```

Comment:
The diff is a little hard to read here, but I'm adding the channel + partner fetches to the `save`.

The basic motivation is that eventually I'm going to get rid of `findOrInsert` which is used primarily for impersonating users (`writer.artsy.net/api/v1/user/:id`). We also use it on article saving, but I don't think we need that -- should just be enough to use `fromAccessToken` since you'll always publish as `me` (`api.artsy.net/api/v1/me`). 

Todo: ensure that a fresh login will save the latest user data